templates/provider/go/app/app.go

```diff
@@ -30,11 +33,21 @@ func NewApp(
 // Start starts the provider process
 func (app App) Start() {
 	// Subscribe
-	err := app.ServiceClient.SubscribeServiceRequest(types.ServiceName, app.RequestCallback)
+	err := app.ServiceClient.SubscribeServiceRequest(app.RequestCallback)
 	if err != nil {
-		app.Logger.Errorf("failed to register service request listener, err: %s", err.Error())
+		app.Logger.Errorf("failed to subscribe service request listener, err: %s", err.Error())
 		return
 	}
 
 	select {}
 }
+
+// Bind provider
+func (app App) Bind(bindConfig servicesdk.BindServiceRequest) {
```

Comment:
why need binding service ?

Reply:
Let's keep it that way for the time being.